cli/src/commands/branch.rs

```diff
@@ -672,10 +676,17 @@ fn cmd_branch_list(
     let mut formatter = ui.stdout_formatter();
     let formatter = formatter.as_mut();
 
-    let branches_to_list = view.branches().filter(|&(name, _)| {
+    let branches_to_list = view.branches().filter(|(name, target)| {
+        let has_conflict = target
+            .local_target
+            .as_normal()
+            .and_then(|id| repo.store().get_commit(id).ok())
+            .map_or(false, |commit| commit.has_conflict().unwrap_or(false));
```

Comment:
`commit.has_conflict()` means the commit content (= trees) has conflict, but I guess "conflict" in #1471 is conflict of branch targets (or diverged branch?)

fyi, the former can be filtered by `-r 'conflict()'`.

Reply:
Ah, I'll rewrite it then.